README.md

```diff
@@ -1,2 +1,25 @@
 Kurtosis Lambda Starter Pack
 ============================
+This repo contains:
+
+1. Example implementations of a Kurtosis Lambda in each language
+1. Infrastructure for bootstrapping a new Kurtosis Lambda, that you can use to create your own customized Kurtosis Lambda
+
+Kurtosis Lambda Quickstart
+--------------------------
+Prerequisites:
+* A [Kurtosis user account](https://www.kurtosistech.com/sign-up)
+* `git` installed on your machine
+* `docker` installed on your machine
+
+Quickstart steps:
+1. Clone this repo's `master` branch: `git clone --single-branch --branch master git@github.com:kurtosis-tech/kurtosis-lambda-starter-pack.git`
+1. View [the supported languages](https://github.com/kurtosis-tech/kurtosis-lambda-starter-pack/blob/master/supported-languages.txt) and choose the language you'd like your Kurtosis Lambda in
+1. Run `bootstrap/bootstrap.sh` and follow the helptext instructions to fill in the script arguments and bootstrap your repo
+1. Customize your own Kurtosis Lambda by editing the generated files inside the `/path/to/your/code/repos/kurtosis-lambda/impl` folder
```

Comment:
Heya I think these would still be better inside the README that gets generated via the bootstrap.sh script. Rationale:
1) When users go away from this document to run `bootstrap.sh` they don't have to come back to this repo to finish the steps
2) If we put it inside their README, it's even more obvious - it'll be in whatever repo they generated.

What we can then do is, at the end of `bootstrap.sh`, say:
```
Bootstrap successful!
1) To build the Lambda, run '/path/to/build.sh'        <-- we fill in this path for them
2) To customize your Lambda, follow the steps in /path/to/generated/README.md   <-- we fill this in for them too
```

Reply:
I decided to put the first part in the Readme of the Starter Pack because users need to do the first part before running the bootstrap file and the other part in the generated readme.
Let me know if you agree with this